packages/builders/src/util/validation.ts

```diff
@@ -0,0 +1,6 @@
+import { setGlobalValidationEnabled } from '@sapphire/shapeshift';
+
+export const enableValidators = () => setGlobalValidationEnabled(true);
```

Comment:
This is not what I said in the RFC 😓. It should disable *just* builder's assertions, not all validators across runtime that use shapeshift

Reply:
oops. Fixed